CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 
-## v2.2.2
-### 2024-07-16
+## v2.3.0
+### 2024-07-17
```

Comment:
This was already released so you'll need a new version above rather than deleting this one.

Reply:
will not do any updates. will let Owen do the next update including the version number
This PR can be closed out as well